engine/src/main/java/org/terasology/logic/ai/AICommands.java

```diff
@@ -25,26 +25,38 @@
 import com.google.common.collect.Iterables;
 
 /**
+ * Artificial Intelligence command
```

Comment:
Really vague. Yes, "AI Commands" naturally contains commands related to AI, spelling that out doesn't really help :-)

In this case I believe the commands all relate to entities that have particular AI components attached, interacting with them somehow.